static/src/js/app/Dashboard.vue

```diff
@@ -9,14 +9,16 @@
     <button type="button" class="btn btn-primary" @click="openModal">
       Quick Add Vocab
     </button>
+    <BookmarkList></BookmarkList>
```

Comment:
Can this element be self-closing?
https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended

Reply:
Absolutely! I didn't know about self-closing components, so thanks for the pointer. I'll fix that.